keys/rsa-private-key.pem

```diff
@@ -0,0 +1,52 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
private key on repo? how fearsome

Reply:
just leaving it here to think, if we want to have that in repository, which is ugly, or do we want to add command generating it to README

Reply:
I literally created a PR with a password to database in the repo a few days ago lol. This repository is named PoC btw

Reply:
didn't see it though, I suppose you want a revert

Reply:
and more seriously, password to db that is not deployed anywhere is meh, but sharing publicly private key that you generated using openssl is a red flag for me

Reply:
one mistake of sharing public pair somewhere by accident (there are no warnings while doing that because it is public) and you are in big trouble

Reply:
why so serious

Reply:
nie no chłopaki nie bijcie się, już zmieniłam pzdr